tutorials/python/python-video-generation/code/Readme.md

```diff
@@ -0,0 +1,60 @@
+# Local environment set up
+---
+
+## Install Python 3
+
+### Linux and OSX
+Using version manager [asdf-vm](https://asdf-vm.com/#/core-manage-asdf-vm?id=install-asdf-vm):
+
+Install [Python Plugin](https://github.com/danhper/asdf-python):
+``` bash
+$ asdf plugin-add python https://github.com/danhper/asdf-python.git
+```
+
+Install Python:
+``` bash
+$ asdf install python 3.6.5
```

Comment:
@Larox You can omit the python version by adding a `.tool-versions`:
```
python 3.6.5
```
Then, you just need: 
```
$ asdf install
```

Reply:
Thanks for the advice! I'll add this :D